rustic.el

```diff
@@ -163,6 +170,22 @@ this variable."
   (when (member mode auto-mode-alist)
     (setq auto-mode-alist (remove mode auto-mode-alist))))
 
+;;; envrc support
+
+;; To support envrc, it is necessary to wrap any buffer creation code
+;; with inheritenv.  Rather than depend on that package, we conditionally
+;; wrap if it is installed.  Users of envrc ought to ensure the inheritenv
+;; package is available before loading rustic.
+
+(defmacro rustic--inheritenv (&rest body)
+  "Wrap BODY so that the environment it sees will match the current value.
+This is useful if BODY creates a temp buffer, because that will
+not inherit any buffer-local values of variables `exec-path' and
+`process-environment'."
+  `(if (featurep 'inheritenv)
```

Comment:
Do you think it makes sense to change it to `(or (featurep 'inheritenv) (require 'inheritenv nil t))` ? Just in case.

Reply:
That doesn't make sense to me.

In order to activate this, prior to loading rustic we must already have done `(require-package 'inheritenv)`, after which `(featurep 'inheritenv)` will be true.

What scenario are you thinking of?